topics/mvvmcross/index.md

```diff
@@ -0,0 +1,12 @@
+---
+aliases: monocross
+created_by: Alexandru Stefan Popescu
+display_name: MvvmCross
+github_url: https://github.com/mvvmcross
+logo: mvvmcross.png
+released: June 2013
+short_description: The .NET MVVM framework for cross-platform solutions, including Xamarin.iOS, Xamarin.Android, Windows and Mac.
```

Comment:
Can you add a comma after "Windows" to suit the style guide?

Reply:
Fixed by more detailed description.

Reply:
Thanks for the detailed description below! However, this line still needs a comma before "and Mac" to suit the style guide.